plugins/modules/ec2_vol.py

```diff
@@ -769,6 +805,9 @@ def main():
     if multi_attach is True and volume_type not in ("io1", "io2"):
         module.fail_json(msg="multi_attach is only supported for io1 and io2 volumes.")
 
+    if wait is True and modify_volume is False:
+        module.fail_json(msg="wait does nothing if modify_volume is False.")
```

Comment:
I guess it should work for now. We would probably also like to add the option to wait for deletion in the future.

Reply:
We are using this module currently and have decided to wait for either `completed` or `optimizing` as waiting for `completed` can take a very long time. Should we do the same here? Or should we have the option to wait for one or more modification states?

How about having an option like this?
```yaml
wait_modification_states:
    description:
      - A list of modification states to wait for. Valid options are "completed" and "optimizing"
    type: list
    default: ["completed", "optimizing"]
```